Detectors/ZDC/calib/src/NoiseCalib.cxx

```diff
@@ -33,7 +33,11 @@ int NoiseCalib::init()
   }
 
   for (int isig = 0; isig < NChannels; isig++) {
-    mH[isig] = new o2::dataformats::FlatHisto1D<double>(4096, -2048.7, 2047.5);
+#ifdef ZDC_NOISECALIB_DATA_SMALL
+    mH[isig] = new o2::dataformats::FlatHisto1D<double>(8191, -2048.25, 2047.25);
```

Comment:
I don't see where do you delete these histos created by new. Why don't you use unique_ptr?